doc/sphinx/source/getting_started/index.rst

```diff
@@ -6,7 +6,7 @@ Getting started
    :maxdepth: 1
 
     Installing ESMValTool <install>
-    User configuration file <config_user>
-    Input data <inputdata>
+    Configuring ESMValTool <config_user>
+    Acquiring input data <inputdata>
     Running ESMValTool <running>
-    Output <outputdata>
+    Inspecting the output <outputdata>
```

Comment:
I like your suggestion for these titles, but then we should change the actual headers accordingly

Reply:
New commit addresses your suggestion.